lib/http/2/stream.rb

```diff
@@ -97,14 +101,33 @@ def receive(frame)
 
       case frame[:type]
       when :data
-        window_size = frame[:payload].bytesize
-        window_size += frame[:padding] || 0
-        @local_window -= window_size
+        # Emit DATA frame
+        delta_size = frame[:payload].bytesize
```

Comment:
frame_size? why "delta"?

Reply:
yup, I "inherited" the naming from `nghttp`.